Get couchUrl dinamically to support multiple ports.

dianabarsan opened the pull request:
Small updates to readme. 
Re-adds ands respects `COUCH_CLUSTER_PORT`.

dianabarsan committed:
get couchUrl dinamically to support multiple ports.

garethbowen reviewed (approved):
Nice cleanup! Thanks.

dianabarsan committed:
fix clustering api not being exposed for medic-os

dianabarsan commented:
I pushed an additional change here. 

Unfortunately the clustering port is not exposed when running against a couchdb running inside the medic-os container. So initializing URLs failed. I changed it so it will only build the clustering API url when specifically needed (it's not needed when running against 3.x couchdb). 
Additionally, I added `NODE_TLS_REJECT_UNAUTHORIZED` env variable, in case medic-os uses a self-signed cert. 

Please let me know if you're still in agreement. 
If yes, I'll merge this, bump the version, publish a new image, and update the docker-compose file as well. 

Thanks!

garethbowen reviewed (approved):
Still good! :D
